docs/custom.css

```
@@ -156,9 +156,12 @@
 
   /* Menu/Sidebar */
   --doc-sidebar-width: 280px;
-  --ifm-menu-color: var(--mastra-text-secondary);
+  --ifm-menu-color: var(--mastra-text-tertiary);
   --ifm-menu-color-active: var(--mastra-text-primary);
   --ifm-menu-link-sublist-icon: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="16px" height="16px" viewBox="0 0 24 24"><path fill="rgba(95,95,95,1)" d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z"></path></svg>');
+  --ifm-menu-link-padding-vertical: 0.5rem;
+  --ifm-menu-color-background-active: rgba(0, 0, 0, 0.05);
```

Stylelint (17.14.0)

[error] Expected "0.05" to be "5%" (alpha-value-notation)

(alpha-value-notation)